finality-aleph/src/runtime_api.rs

```diff
@@ -115,10 +154,80 @@ where
             return Ok(authorities);
         }
 
-        let queued_keys: QueuedKeys = self.read_storage("Session", "QueuedKeys", at)?;
+        let queued_keys: QueuedKeys = self.read_storage_value("Session", "QueuedKeys", at)?;
         Ok(queued_keys
             .into_iter()
             .filter_map(|(account_id, keys)| keys.get(AURA).map(|key| (account_id, key)))
             .collect())
     }
 }
+
+#[cfg(test)]
+mod test {
+    use std::{
+        collections::{BTreeMap, HashMap},
+        sync::Arc,
+    };
+
+    use frame_support::Twox128;
+    use parity_scale_codec::Encode;
+    use sp_runtime::Storage;
+    use substrate_test_client::ClientExt;
+
+    use super::*;
+    use crate::testing::mocks::{TestClientBuilder, TestClientBuilderExt};
+
+    #[tokio::test]
+    async fn test_storage_reads() {
```

Comment:
Maybe we can add a test which uses `Twox64Concat`, as there are `StorageMaps` on our chain that uses those

Reply:
Also `at_block` and some errors are not tested, maybe it's worth having such tests, as storage reads are quite an important functionality?

Reply:
Good point, I'll add another tests

Reply:
done